backend/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:12.13-alpine as development
```

Comment:
Does this dockerfile create two different images (development and produciton) from the same build?